src/main/java/com/epam/esm/SpringConfig.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.esm;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.LocaleResolver;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.i18n.CookieLocaleResolver;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import javax.servlet.http.HttpServletRequest;
+import javax.sql.DataSource;
+import java.util.Locale;
+
+@SpringBootApplication
+@EnableWebMvc
+@EnableTransactionManagement
+@PropertySource("classpath:application.properties")
+public class SpringConfig implements WebMvcConfigurer {
```

Comment:
In Sping Boot you should not have beans like `jdbcTemplate`, `transactionManager`, etc. Most of the things should be configured automatically

Reply:
All the unnecessary bean declarations removed

Reply:
@a1exanddrovich all the beans are still present